### PR TITLE
Made improvements to the 2 rules on Power BI Version Control

### PR DESCRIPTION
**Pull request by divgul**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

✏️ Review by @adamcogan and @calumjs 

> 2. What was changed?

✏️- Images containing client details were replaced
     - Changes in headings, captions, and textual emphases to make the rule easier to understand

> 3. Did you do pair or mob programming (list names)?

✏️No

### CODE REVIEW

File: rules/do-you-use-version-control-with-power-bi/rule.md [hunk 1]
```diff
@@ -40,73 +40,75 @@ In other words, the history of the changes isn't recorded anywhere.
...
-You should no longer edit or publish reports directly in the production workspace on the Power BI Service. A better process for editing and committing reports is described below.
-
-### Business Users
+### Business Users (Use Power BI Service)
 
 If you're a business user, watch the following video to get a walkthrough of the process you would follow to edit and commit reports.
 
 `youtube: https://www.youtube.com/watch?v=dlOK6QBEyQo`
-**Video: Power BI Source Control for the Business User**
+**Video: Power BI Source Control for the Business User (timebox 8:50 mins)**
```
Comment (calumjs) [inline rules/do-you-use-version-control-with-power-bi/rule.md:73]: I think this should just say (8 mins) after the video name
as per the rule https://www.ssw.com.au/rules/right-format-to-write-videos-time-length/

File: rules/do-you-use-version-control-with-power-bi/rule.md [hunk 2]
```diff
@@ -40,73 +40,75 @@ In other words, the history of the changes isn't recorded anywhere.
...
 If you want to update the report's data model or want more sophisticated editing features, you will need to edit the report in Power BI Desktop instead. The next section explains how you can do so.
 
-### Developers
+### Developers (Use Power BI Desktop)
 
 If you're a developer, watch the following video to get a walkthrough of the process you would follow to edit and commit reports.
 
 `youtube: https://www.youtube.com/watch?v=MpedXah-Hv0`
-**Video: Power BI Source Control for Developers**
+**Video: Power BI Source Control for Developers (timebox 5:00 mins)**
```
Comment (calumjs) [inline rules/do-you-use-version-control-with-power-bi/rule.md:94]: I think this should just say (5 mins) after the video name
as per the rule https://www.ssw.com.au/rules/right-format-to-write-videos-time-length/

File: rules/do-you-use-version-control-with-power-bi/rule.md (hunk 1 above)
Comment (divgul) [reply rules/do-you-use-version-control-with-power-bi/rule.md]: Done

File: rules/do-you-use-version-control-with-power-bi/rule.md (hunk 2 above)
Comment (divgul) [reply rules/do-you-use-version-control-with-power-bi/rule.md]: Done

**Review by calumjs — APPROVED** [review]

**Pull request merged**